lib/frodo/service.rb

```diff
@@ -172,6 +172,10 @@ def logger
       end
     end
 
+    def with_metadata?
+      !options[:with_metadata] || options[:with_metadata]
```

Comment:
So the default for this is to be on?

Reply:
I approved this, because I don't think this is blocking / it's not a one-way door and you can obviously set the value to whatever you choose.

My thought for the default being false is that the `with_metadata? # => true` means a lot more work is going to be done out of the box. It seems like something you would enable rather than have to explicitly disable. But this is an opinion.

Again don't think this needs to block the PR, can even end up as a different task if others agree and someone can update it later. Or people don't agree and it's fine :D